src/androidTestUnbrandedWebviewDebug/java/org/medicmobile/webapp/mobile/SettingsDialogActivityTest.java

```diff
@@ -121,7 +123,7 @@ public void testLoginScreen() throws Exception {
 				.perform(webClick());
 		Thread.sleep(4000);//TODO: use better ways to handle delays - takes longer with emulators
 		onWebView().withElement(findElement(Locator.CSS_SELECTOR, "p.error.incorrect"))
-				.check(webMatches(getText(), containsString(ERROR_INCORRECT)));
+				.check(webMatches(getText(), matchesPattern(ERROR_INCORRECT)));
```

Comment:
Is it possible to select the language in the login page instead? 
Because if we decide to add more gamma testing, it's going to be difficult to maintain 2 labels every time.